Makefile

```diff
@@ -20,6 +20,9 @@ sync_undirected_graph_leader_example:
 sync_undirected_mesh_leader_example:
 	go run example/sync_undirected_mesh_peterson_leader.go 6 9
 
+sync_bgp_example:
+	go run example/sync_bgp_phase_king.go 10 3 0 1 0 1 0 1 1 0 0 0
```

Comment:
Just for consideration: maybe instead of long list of 0s and 1s use a single bigint number encoding it? See https://pkg.go.dev/math/big